lib/Backend/amd64/LinearScanMD.h

```diff
@@ -37,7 +37,6 @@ class LinearScanMD : public LinearScanMDShared
     void        GenerateBailOut(IR::Instr * instr,
                                 __in_ecount(registerSaveSymsCount) StackSym ** registerSaveSyms,
                                 uint registerSaveSymsCount);
-    IR::Instr* GenerateBailInForGeneratorYield(IR::Instr* resumeLabelInstr, BailOutInfo* bailOutInfo);
```

Comment:
What did this method do before?

Reply:
It was a draft Jit implementation for resuming a generator function. Whilst jitting generators is still not enabled the current implementation no longer uses this code.

Reply:
Can some of it be reused in the future, when we enable that?

Reply:
The Generator Jit implementation whilst still not enabled was re-written almost completely by @nhat-nguyen and further edited pretty significantly by @zenparsing - the current version works except when it hits a bailout in the wrong place which is a bug we need to solve before enabling it.

This bit of code is left over from the first implementation prior to the work the two of them did - it's not compatible with what they've done and is just left as noise in the source tree.